### PR TITLE
feat(windows): add nucleus bootstrap script

### PR DESCRIPTION
**Pull request by shaguptashaikh**
**Issue #, if available:**
-

**Description of changes:**
Adding bootstrap script for Windows to nucleus recipe

**Why is this change necessary:**
Required for OTA to work on Windows

**How was this change tested:**
Related acceptance test on Windows passes

**Any additional information or context required to review the change:**

**Checklist:**
 - [ ] Updated the README if applicable
 - [ ] Updated or added new unit tests
 - [ ] Updated or added new integration tests
 - [ ] Updated or added new end-to-end tests
 - [ ] If your code makes a remote network call, it was tested with a proxy
 - [ ] You confirm that the change is backwards compatible

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: conf/recipe.yaml [hunk 1]
```diff
@@ -55,3 +55,24 @@ Manifests:
           exit 100
   - Platform:
       os: windows
+    Artifacts:
+      - URI: s3://use-local-provided-artifact/custom.zip
+        Unarchive: zip
+        Permission:
+          execute: ALL
+          read: ALL
```
Comment (MikeDombo) [inline conf/recipe.yaml:63]: remove this, we don't have it for other platforms

File: conf/recipe.yaml [hunk 2]
```diff
@@ -55,3 +55,24 @@ Manifests:
...
+    Lifecycle:
+      bootstrap:
+        RequiresPrivilege: true
+        script: |-
+
+          powershell -command "Set-StrictMode -Version Latest;"^
```
Comment (MikeDombo) [inline conf/recipe.yaml:69]: you sure that this works? How did you test it?
My experience is that multi-line commands won't work (despite escaping with `^`).

File: conf/recipe.yaml (hunk 1 above)
Comment (shaguptashaikh) [reply conf/recipe.yaml]: Oops, removed

File: conf/recipe.yaml (hunk 2 above)
Comment (shaguptashaikh) [reply conf/recipe.yaml:63]: Yes I've tested like the description says

**Review by tilo-chen — DISMISSED** [review]

**Review by MikeDombo — APPROVED** [review]

**Review by tilo-chen — APPROVED** [review]

**Pull request merged**